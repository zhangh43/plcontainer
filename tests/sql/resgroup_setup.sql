create resource group plgroup with (concurrency=0, cpu_rate_limit=10, memory_limit=30, memory_auditor=1);
create resource group plgroup2 with (concurrency=0, cpu_rate_limit=10, memory_limit=10, memory_auditor=1);
