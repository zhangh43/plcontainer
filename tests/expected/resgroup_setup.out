alter resource group admin_group set memory_limit 35;
create resource group plgroup with (concurrency=0, cpu_rate_limit=10, memory_limit=35, memory_auditor=1);
